build/make/deploy-debian.mk

```diff
@@ -56,8 +58,8 @@ else
 endif
 
 .PHONY: undeploy
-undeploy: deploy-check remove-package-from-repo publish
+undeploy: deploy-check remove-package-from-repo publish ## Undeploy package from apt repository
 
 .PHONE: lint-deb-package
-lint-deb-package: debian
+lint-deb-package: debian ## Build and lint debian package
```

Comment:
Because of the way how `make` works you can leave out `Build and`. If we had build a debian package beforehand, `make` should lint it without building it again.